api/routes/session.py

```diff
@@ -22,6 +22,8 @@ def start_session(user_id):
     db.session.execute(del_sess)
     del_slot = delete(Slot).where(Slot.user_id == user_id)
     db.session.execute(del_slot)
+
+    # TODO: only delete uncompleted tasks
```

Comment:
this shouldn't be particularly hard TODO to implement, just change where to also check that `Task.user_id.completed` is true OR `Task.user_id.completed_in_slot` is not null

Reply:
Yes, I just haven't gotten around to it yet